### DISCUSSION TITLE
serverDependenciesToBundle now works?!

### DISCUSSION THREAD

**Discussion by silvenon** [remix-run/remix#3247] [General]
Hey, @chaance, one fix that went under the radar in 1.5.0 that I think should be mentioned in the release notes is that importing ESM-only dependencies now works fine without having to do any dynamic import acrobatics! 🎉 Not only that, but Remix tells you about every single dependency during development mode that should be added to `serverDependenciesToBundle`, I don't recall that working correctly before. 🤔 

I'm sure that for many this is additional cause for big celebration, I'm just not sure what's the story behind that.

**Comment by chaance** [comment]
@silvenon thanks for the heads up! Do you happen to know the PR or commit associated with that fix?

**Reply by silvenon** [reply]
Unfortunately no, I just know that after upgrading from 1.4.3 to 1.5.0 Remix started warning me about dynamically imported ESM-only dependencies.

Maybe the server bundling feature itself always worked, but since 1.5.0 Remix is able to warn about _every_ ESM-only dependency (and dependencies of dependencies), and it didn't work for me before only because I failed to list all of the necessary dependencies.

I hope this info is at least somewhat useful to your team. 🤞
